### ISSUE TITLE
Don't store the subject of the entry in the bookmark table

### ISSUE THREAD

**Issue by auge8472** [ilosuna/mylittleforum#134]
If one corrects the subject of an entry after someone added the entry to hers or hims bookmark list, the stored subject says the old one and not the corrected. To read the subject *directly* from the entries-table instead, is the way to go (IMHO).

**Comment by derletztekick** [comment]
No, that was not my intention. Often, the posting author din't change the topic. So, the title is featureless. In bookmark list, the user can rename the topic by a more specific one -  to get to the point of the posting independent of the (current) subject.

**Comment by derletztekick** [comment]
> It's (together with the authors name and the entry date) an identifier for the entry itself.

In worst case, it is the featureless topic of the thread opener, that not reflect the bookmarked content. Check your bookmark list in your browser. You can change the topic in what ever you want e.g. nice CSS hack for feature XY.
